cmd/command.go

```diff
@@ -594,6 +595,34 @@ func hIncrBy(cli *Client, args [][]byte) (interface{}, error) {
 	return cli.db.HIncrBy(key, field, incrInt64Val)
 }
 
+func hRandField(cli *Client, args [][]byte) (interface{}, error) {
+	if len(args) < 1 || 3 < len(args) {
```

Comment:
!= 1 || !=3

Reply:
Sorry, something is wrong.
The correct was as follows.

`if len(args) != 1 && len(args) != 2 && len(args) != 3 {`